sdk/mobile-center/src/test/java/com/microsoft/azure/mobile/utils/ShutdownHelperTest.java

```diff
@@ -0,0 +1,46 @@
+package com.microsoft.azure.mobile.utils;
+
+import android.os.Process;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.rule.PowerMockRule;
+
+import static org.junit.Assert.assertNotNull;
+import static org.powermock.api.mockito.PowerMockito.mockStatic;
+import static org.powermock.api.mockito.PowerMockito.verifyStatic;
+import static org.powermock.api.mockito.PowerMockito.when;
+
+@SuppressWarnings("unused")
+@PrepareForTest({ShutdownHelper.class, Process.class})
```

Comment:
Disagree. Without this, test not work properly: call `ShutdownHelper.shutdown` killing the test process.